gateway-home/modules/ROOT/pages/gateway-overview.adoc

```diff
@@ -6,167 +6,48 @@ endif::[]
-| Choose Local Mode to mostly disconnect from the control plane. Manage Flex Gateway in Local Mode with locally stored declarative configuration files.
-
-| *Policy Application*
-| Use API Manager to apply policies in Connected Mode.
-| Apply policies in Local Mode via local declarative configuration files.
-
-| *Air-Gapped?*
-| No. Flex Gateway is connected to the control plane.
-| No. Flex Gateway is managed locally but connected to the control plane for registration and usage metrics.
-
-|===
-
-=== Installation Options
-
-You can install Flex Gateway in a variety of ways:
-
-* A standalone single runtime or replica on various Linux environments including Ubuntu, Debian, Amazon Linux, RHEL, and CentOS.
-
-* A standalone in a Docker container.
-
-* An Ingress controller in Kubernetes, for highly-available, high-performance use cases.
-
-* A sidecar to a backend application in a Docker container, thereby protecting a single backend application.
-
-In general, if the use case involves protecting a few APIs and the number of transactions are low, with a minimal impetus on high-availability or robustness, choose to install as a standalone gateway. Otherwise, choose to install as an Ingress controller in Kubernetes.
+Anypoint Flex Gateway is an ultrafast lightweight API gateway designed to manage and secure APIs running anywhere.
+Built to seamlessly integrate with DevOps and CI/CD workflows, Anypoint Flex Gateway delivers the
+performance required for the most demanding applications and microservices while providing enterprise
+security and manageability across any environment.
 
 For more information, see xref:gateway::flex-gateway-getting-started.adoc[Getting Started with Flex Gateway].
 
 == Anypoint Mule Gateway
 
-While Flex Gateway is a high-performance Envoy-based gateway with a small footprint, MuleSoft also provides a simple Mule Gateway for Mule Applications.
-
-The Mule Runtime, based on Java Spring, includes an embedded Mule Gateway. By using this gateway, you can, for example:
-
-* Apply a basic authentication policy on top of a Mule application.
-* Enrich an incoming or outgoing message.
-* Add any other complex capability to an API without having to write any code.
+Mule Runtime includes an embedded Mule Gateway. Using this gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
 
-Mule Gateway protects a single Mule API. The key advantage is that for application developers building Mule applications, basic endpoint protection becomes trivial.
+Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
```

Comment:
Remove "Overall"?
enables?

What about making the second sentence the part of the intro and putting the example after?

Mule Runtime includes....Mule Gateway enables you to...

Using Mule Gateway, any user can, for example.....

Reply:
Fixed